Makefile

```diff
@@ -4,7 +4,7 @@ EMBASSY_SRC := raspios.img product_key.txt $(EMBASSY_BINS) backend/embassyd.serv
 COMPAT_SRC := $(shell find system-images/compat/src)
 UTILS_SRC := $(shell find system-images/utils/Dockerfile)
 BACKEND_SRC := $(shell find backend/src) $(shell find patch-db/*/src) $(shell find rpc-toolkit/*/src) backend/Cargo.toml backend/Cargo.lock
-FRONTEND_SRC := $(shell find frontend/projects) $(shell find frontend/styles) $(shell find frontend/assets)
+FRONTEND_SRC := $(shell find frontend/projects) $(shell find frontend/assets)
```

Comment:
Why is this in here?

Reply:
running `make` to ensure this still built for prod revealed an error thrown that this directory was missing. since it no longer exists, i deleted.